src/util/rocksdb/properties.rs

```diff
@@ -466,6 +467,188 @@ impl DecodeProperties for UserCollectedProperties {
     }
 }
 
+#[derive(Debug, Clone, PartialEq, Eq, Copy)]
+pub enum OffsetKindInRange {
+    Size,
+    Keys,
+}
+
+#[derive(Debug, Default, Clone)]
+pub struct OffsetsInRange {
```

Comment:
“XXXInRange" looks weird as a struct name, how about "RangeOffsetKind" and "RangeOffsets"?